app/scripts/controllers/permissions/index.js

```diff
@@ -133,7 +133,8 @@ class PermissionsController {
   /**
    * User approval callback. The request can fail if the request is invalid.
    *
-   * @param {object} approved the approved request object
+   * @param {object} approved - the approved request object
```

Comment:
I'm not sure how to import the [`IPermissionsRequest` type](https://github.com/MetaMask/rpc-cap/blob/master/src/%40types/index.d.ts#L21) from rpc-cap here :thinking:  I left it as `object` for now.